lib/asciidoctor/stylesheets.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# Monkey patch the whole file because the class is static
```

Comment:
@mojavelinux Do you think we can do something in Asciidoctor core to make it easier to override ?